application/modules/quotes/models/Mdl_quotes.php

```diff
@@ -217,13 +217,13 @@ public function copy_quote($source_id, $target_id)
 
         // Copy the custom fields
         $this->load->model('custom_fields/mdl_quote_custom');
-        $db_array = $this->mdl_quote_custom->where('quote_id', $source_id)->get()->row_array();
+        $custom_fields = $this->mdl_quote_custom->where('quote_id', $source_id)->get()->result();
```

Comment:
@naui95, because I wanted to use the copy quotes function in IP I discovered this problem. Although I am absolutely no programmer I tried to solve this bug by comparing this "**_copy quotes_**" function with the "**_copy invoices_**" function. 
I succeeded by copying the piece of code from the "**_copy invoices_**" function. 
Your proposal certainly seems to me to be a better solution programmatically.
Therefore, isn't it better and desirable to modify your proposal/solution in the "**copy invoices**" function as well?

Reply:
Hi @Verony-makesIT thank you 🙏. Yes, you might be right. I did not look at how the copying with invoices is done. I will check it out this weekend. If the logic should be the same it might be wise to adapt also the copy invoice as you are suggesting. I will let you know.